templates/allauth/socialaccount/snippets/slack_signin.html

```diff
@@ -0,0 +1,12 @@
+{% load socialaccount %}
+
+{% get_providers as socialaccount_providers %}
+
+{% for provider in socialaccount_providers %}
+    {% if provider.name == 'Slack' %}
+    <a title="{{ provider.name }}" class="socialaccount_provider {{ provider.id }}"
+        href="{% provider_login_url provider.id process=process scope=scope auth_params=auth_params %}">
+        <img src="https://api.slack.com/img/sign_in_with_slack.png" />
```

Comment:
`<img....>` 😉  not `<img..... />`

Reply:
Changed this as well. (I copied this from the Slack site btw haha)